core/types/l2trace.go

```diff
@@ -147,54 +145,6 @@ type StorageWrapper struct {
 	Value string `json:"value,omitempty"`
 }
 
-type TransactionData struct {
```

Comment:
suggest keep `TransactionData` and add `maxPriorityFeePerGas` & `maxFeePerGas`,
because `ChainId` & `IsCreate` is useful in libzkp

Reply:
This PR is a breaking change. discussed with @lispc before. An example:
https://github.com/scroll-tech/go-ethereum/pull/622/files#diff-b8286c818aa670d24325dd2506ae98792e2952165dbf70e716ffadcb77dbb41b
some formats have changed...

Reply:
i see. if @lispc confirms that we don't need chainid in tx any more we can adapt this change

Reply:
yeah. and there are other changes, like from `decimal` to `hex`. thus we can double-check that and test it if these changes are acceptable.

Reply:
i remember chain_id exists in block level?  we will be ok if chain_id is remove in tx level.

Reply:
block level still has chainID: https://github.com/scroll-tech/go-ethereum/blob/scroll-v5.1.13/rollup/tracing/tracing.go#L535

Reply:
I come with an additional question, would the circuit's "parser" (idk how to call it) both support old and new l2 trace simultaneously? thus the l2geth can be upgraded seamlessly.

a case without downtime:
1. upgrade `circuit`.
2. upgrade `rollup-relayer` (this would be backward compatible).
3. then finally upgrade `l2geth`.